ui/tests/acceptance/pki/pki-engine-workflow-test.js

```diff
@@ -379,7 +379,7 @@ module('Acceptance | pki workflow', function (hooks) {
       await visit(`/vault/secrets/${this.mountPath}/pki/overview`);
       await click(SELECTORS.issuersTab);
       assert.dom('[data-test-serial-number="0"]').exists({ count: 1 }, 'displays serial number tag');
-      assert.dom('[data-test-common-name="0"]').exists({ count: 1 }, 'displays cert common name tag');
+      assert.dom('[data-test-common-name="0"]').doesNotExist('does not display cert common name tag');
```

Comment:
I tested this locally with the given policy in both the current UI and in the shipped binary, and each time the issuer read failed and I wasn't able to see the common name. We learned with the kv workflow tests that the store doesn't always clear between logout/login so maybe this was working due to that?